### PR TITLE
Top contributors

### PR DESCRIPTION
**Pull request by agriffard**
Add top 20 contributors in Contributors.md

### CODE REVIEW

File: Contributors.md [hunk 1]
```diff
@@ -12,11 +12,50 @@ their names indented and be marked with a '-'
...
+
+* andy (@infofromca)
+
+* Carl Woodhouse (@carlwoodhouse)
+
+* Lukas Kabrt (@lukaskabrt)
+
+* Siyamand Ayubi (@siyamandayubi)
+
+* Peter Davis (@petedavis)
```
Comment (sebastienros) [inline Contributors.md:51]: duplicates

**Comment by sebastienros** [comment]
Top 150 ?

**Comment by hishamco** [comment]
I think this https://allcontributors.org/ will automate the process

**Comment by agriffard** [comment]
Sorry, it has an auto-commit feature.

Test bot: 
@all-contributors please add agriffard for doc

**Comment by hishamco** [comment]
Seems the bot start working ;)

**Comment by agriffard** [comment]
https://github.com/OrchardCMS/OrchardCore/blob/ag/contributors/Contributors.md

**Comment by agriffard** [comment]
@sebastienros 154 contributors added.

**Pull request merged**